conjur-oss/README.md

```diff
@@ -145,7 +145,7 @@ $  CONJUR_NAMESPACE=my-conjur-namespace
 $  kubectl create namespace "$CONJUR_NAMESPACE"
 $  DATA_KEY="$(docker run --rm cyberark/conjur data-key generate)"
 $  HELM_ARGS="--set dataKey=$DATA_KEY \
-              --set image.tag=1.5.1 \
+              --set image.tag=1.11.0 \
```

Comment:
1.10?

Reply:
This was intentional. It's an example for Helm installing with a specific version of Conjur, so I wanted to use something non-default, and something recent.